packages/wrangler/src/versions/index.ts

```diff
@@ -254,5 +257,12 @@ export default function registerVersionsSubcommands(
 			"Safely roll out new Versions of your Worker by splitting traffic between multiple Versions [beta]",
 			versionsDeployOptions,
 			versionsDeployHandler
+		)
+		.command(
+			"secret",
+			"🤫 Generate a secret that can be referenced in a Worker",
```

Comment:
```suggestion
			"Generate a secret that can be referenced in a Worker",
```

Reply:
because we're not doing emojis for sub-commands anymore, as per https://github.com/cloudflare/workers-sdk/pull/3735